cli.py

```diff
@@ -61,20 +61,28 @@ def update_normalizer_db(normalizer, dev, db_url, update_all):
             normalizers = list(src for src in sources)
             CLI()._update_normalizers(normalizers, sources, db)
         elif not normalizer:
-            ctx = click.get_current_context()
-            click.echo("Must either enter 1 or more sources, or use `--update_all` parameter")  # noqa: E501
-            click.echo(ctx.get_help())
-            ctx.exit()
+            CLI()._help_msg()
         else:
             normalizers = normalizer.lower().split()
 
+            if len(normalizers) == 0:
+                CLI()._help_msg()
+
```

Comment:
is it possible for code to get here? if no parameter is given for `normalizer`, or if it's called with an empty string (like `normalizer=''`) then it's evaluated as None (and follows the branch in line 63-64).